analytics/analytics.go

```diff
@@ -24,6 +24,8 @@ type Latency struct {
 	Upstream int64 `json:"upstream"`
 }
 
+const SQLTABLE = "tyk_analytics"
```

Comment:
In Golang, const naming convention is like: `SQLTable` and `UptimeSQLTable`. I will approve it anyway but wanted to mention if you'd like to change.

Reply:
Done!